.babelrc

```diff
@@ -19,6 +19,8 @@
       }
     ],
     "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-optional-chaining",
```

Comment:
Suprised this wasn't already in! We can no do 
```
const deepPropOrUndefined = foo?.bar?.baz?.bam
```
which should eliminate our need for the `get` util.

Reply:
oo sick!